### DISCUSSION TITLE
Why does the dreambooth extension tab not appear after a couple of times that I use the same colab?

### DISCUSSION THREAD

**Discussion by MasterDenis** [TheLastBen/fast-stable-diffusion#1639] [Q&A]
Basically what the title says, I have some logs and screenshots here of the problem, this is of course after having restarted both the ui and the cell.

![Screenshot 2023-02-27 045203](https://user-images.githubusercontent.com/61897713/221462344-40370544-dc44-4a69-9ee1-f668010092a8.png)

![Screenshot 2023-02-27 045320](https://user-images.githubusercontent.com/61897713/221462353-7c4bfe28-1434-4638-893c-2d42e2e7355b.png)

PS: ControlNet also doesnt work from either txt2img nor img2img (it only works if put on the main img2img image import slot)

Thanks

**Comment by meunumerotim333** [comment]
Another person said here that ControlNET problem is a browser issue, it works well on Chrome.

**Reply by MasterDenis** [reply]
I use Chrome!

**Comment by TheLastBen** [comment]
Dreambooth extension require different dependencies than the default ones packaged with fast stable diffusion, so it's not compatible.
